app/faucet/templates/shared/faucet_network_status.html

```diff
@@ -0,0 +1,24 @@
+{% comment %}
+    Copyright (C) 2017 Gitcoin Core
+
+    This program is free software: you can redistribute it and/or modify
+    it under the terms of the GNU Affero General Public License as published
+    by the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    This program is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+    GNU Affero General Public License for more details.
+
+    You should have received a copy of the GNU Affero General Public License
+    along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+{% endcomment %}
+                            <div id=network_status class="w-100" style="text-align: center;font-size: smaller;">
+                                <div class="w-40">
+                                        Web3: <span>☑ - Enabled</span>
+                                        <br>
+                                        Eth: <span>☑ - About Zero</span>
```

Comment:
> About Zero

:)

Reply:
Seems correct.... we are looking for potential contributors with zero or almost zero eth